lib/solar_system.rb

```diff
@@ -0,0 +1,29 @@
+class SolarSystem
+  attr_reader :planets
+
+  def initialize (star_name)
+    @star_name = star_name
+    @planets = [] 
+  end
+
+  def add_planets(some_planet)
+    @planets << some_planet
+  end
+
```

Comment:
Since this only adds one planet at a time, I would probably change the name to `add_planet`.